lib/src/transformer/impl_generation.dart

```diff
@@ -402,10 +402,6 @@ class ImplGenerator {
             annotations.Accessor accessorMeta = instantiateAnnotation(field, annotations.Accessor);
```

Comment:
Yeah, I don't think this will pick up the constants.

You'll probably have to just match them by name, like so:
```dart

T getConstantAnnotation<T>(AnnotatedNode member, String name, T value) =>
    member.metadata.any((annotation) => annotation.name?.name == name)) ? value : null;


annotations.Accessor requiredProp = instantiateAnnotation(field, 'requiredProp', annotations.requiredProp);
annotations.Accessor nullableRequiredProp = instantiateAnnotation(field, 'nullableRequiredProp', annotations.nullableRequiredProp);

if (requiredProp != null) {
  // ...
}
if (nullableRequiredProp != null) {
  // ...
}
```